Remotion/Data/DomainObjects.UnitTests/ReflectionUtilityTest.cs

```diff
@@ -138,6 +142,14 @@ public void IsInheritanceRoot_BaseTypeIsDomainObject ()
       Assert.That(ReflectionUtility.IsInheritanceRoot(type), Is.True);
     }
 
+    [Test]
+    public void IsInheritanceRoot_SupportsIDomainObjectInterfaces ()
+    {
+      var type = typeof(ITestDomainObject);
+
```

Comment:
Please add the same checks and test-naming as is done in `IsInheritanceRoot_BaseTypeIsDomainObject`. Best to rename the class-based tests to include "class" or "DomainObject" and the interface-based tests to include "interface" or "IDomainObject".

e.g. IsInheritanceRoot_WithClass_AndBaseTypeIsDomainObject_ReturnsTrue (added the returns-suffix since that's good practice anyway)

Also, please add a test for a derived domainobject that introduces interface to show that this isn't an inheritance root. 

There's also more inheritance root tests for derived classes. please check those and add the appropriate itnerface cases, too.